lib/pry-remote-em/sandbox.rb

```diff
@@ -0,0 +1,40 @@
+module PryRemoteEm
+  class Sandbox
```

Comment:
A short description for the purpose of `Sandbox` would be helpful for sleepy people like me. :)

Reply:
I agree, I added a big paragraph in Readme!